charmcraft.yaml

```diff
@@ -103,6 +103,15 @@ config:
         Run Kratos on dev mode, it is needed if HTTPS is not set up. This should only be used for development purposes.
       type: boolean
       default: False
+    enable_local_idp:
+      description: Enable Kratos Identity Provider
+      type: boolean
+      default: True
+    enable_passwordless_login_method:
+      description: |
+        Enable login with passwordless webauthn method. Requires `enable_local_idp=True`.
```

Comment:
suggestion: would `webauthn` be a better word than `passwordless`? IMO passwordless is too generic, but I have no problem with keeping it as it is if you feel this is easier to understand

Reply:
I thought webauthn main use case is passwordless login, eg use your yubikey instead of having to remember a bunch of passwords. `passwordless` is probably more user friendly for users that have little experience with identity.